tests/php/ORM/DataListTest.php

```diff
@@ -1528,6 +1578,64 @@ public function testExcludeOnFilter()
             $sql
         );
         $this->assertEquals(array('Phil is a unique guy, and comments on team2', 'Bob'), $parameters);
+        $this->assertListEquals([['Name' => 'Phil']], $list);
+    }
+
+    /**
+     * Test that if a complicated exclude() is applied to a filter(), the filter() is still preserved.
+     */
+    public function testComplicatedExcludeOnFilter()
+    {
+        /**
+         * @var DataList $list
```

Comment:
Can this be an inline comment like `/** @var DataList $list */` please

Reply:
This is already a typehint on `DataObject::get()` isn't it?